bot/modules/owner.py

```diff
@@ -113,15 +114,17 @@ def __init__(self, bot):
             await ctx.send('**`SUCCESS`**')
             logger.info('<{0}> UnLoaded successfully.'.format(cog))
 
-    @commands.command(name='reload', hidden=True)
+    @commands.command(name='reload')
     @commands.is_owner()
     async def cog_reload(self, ctx, *, cog: str):
         """Command which Reloads a Module.
         Remember to use dot path. e.g: cogs.owner"""
 
+        cogfix = 'modules.'+cog
+
         try:
-            self.bot.unload_extension(cog)
-            self.bot.load_extension(cog)
+            self.bot.unload_extension(cogfix)
+            self.bot.load_extension(cogfix)
```

Comment:
remember to fix this to full_cog_path if we go with the change
```suggestion
            self.bot.load_extension(full_cog_path)
```